cloud/pod_util.go

```diff
@@ -107,3 +128,32 @@ func podAWSOptions(settings *evergreen.Settings) awsutil.ClientOptions {
 
 	return *opts
 }
+
+// toEnvVars converts a map of environment variables and a map of secrets to a slice of cocoa.EnvironmentVariables.
+func toEnvVars(envVars map[string]string, secrets map[string]string, secretIDs []string) []cocoa.EnvironmentVariable {
+	allEnvVars := []cocoa.EnvironmentVariable{}
```

Comment:
`var envVars []cocoa.EnvironmentVariable` for the same reason I mentioned in a previous PR about initializing an empty vs nil slice and then `append`ing to it.